gordo/machine/dataset/data_provider/ncs_reader.py

```diff
@@ -346,13 +346,16 @@ def read_tag_files(
                 if dry_run:
                     logger.info("Dry run only, returning empty frame early")
                     return pd.Series()
-
+                before_downloading = timeit.default_timer()
                 with adls_file_system_client.open(file_path, "rb") as f:
                     df = file_type.read_df(f)
                     df = df.rename(columns={"Value": tag.name})
+                    df.index = pd.to_datetime(df.index, utc=True)
```

Comment:
This looks a bit weird for the csv case now, with double converting

Reply:
I am not sure in which date format in the current parquet files in datalake, if you think this is a double conversion, I'd rather have it than not.

Reply:
The point is that the way it was before was that `file_type.read_df(f)` handles all conversion including time parsing. Now it still does that for the csv case (and then parses again here) while for the parquet it's only here. You should either handle the timeseries parsing inside `read_df(f)` for both the csv and parquet readers, or in none of them.

Reply:
You have a good point, moved to ParqutFileType.